templates/zerver/login.html

```diff
@@ -104,7 +104,7 @@ <h1 class="get-started">{{ _("Log in to Zulip") }}</h1>
                             </button>
                         </form>
 
-                        {% if any_social_backend_enabled %}
+                        {% if page_params.external_authentication_methods|length > 0 %}
```

Comment:
I suspect we have the same bug in `accounts_home.html`, and can then delete `any_social_backend_enabled` from `zerver/context_processors.py`.

Reply:
@timabbott: That should be fixed now, can you please approve the workflows.